docs/_docs/04_reference/cli-options.md

```diff
@@ -501,6 +502,29 @@ for all of your Terraform modules, and for each module processed by the `xxx-all
 append the path of `source` parameter in each module to the `--terragrunt-source` parameter you passed in.
 
 
+### terragrunt-source-map
+
+**CLI Arg**: `--terragrunt-source-map`<br/>
+**Requires an argument**: `--terragrunt-source-map git::ssh://github.com=/path/to/local-terraform-code`
+
+Can be supplied multiple times: `--terragrunt-source-map source1=dest1 --terragrunt-source-map source2=dest2`
+
+The `--terragrunt-source-map source=dest` param replaces any `source` URL (including those defined in `dependency`
+blocks) that has root `source` with `dest`.
```

Comment:
When you say in `dependency` block, do you mean the `config_path`? Or a `source` URL _within_ `dependency`?

Reply:
Source URL within dependency. Updated docs to clarify this: 44de255ddc0b249f8cf7584dab8d25e6505c9742

Reply:
Roger, thanks!